_templates/Additional_sidebar.html

```diff
@@ -0,0 +1,31 @@
+<h1>Welcome To my page </h1>
+<body><h2>My name is Enmanuel Morillo </h2>
+  <p>I'm majoring in <em>COMPUTER SCIENCE !</em> </body>
+
+<style>
+  em{ color: hsl(308, 72%, 76%);font-size: medium;}
```

Comment:
it will be easier to maintain in the future if you put this in the custom.css file

Reply:
ok thank you